Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -514,7 +514,10 @@ class BocList
     var tableBlock = bocList.querySelector(':scope > div.bocListTableBlock')!;
 
     var scrollTimer: Nullable<number> = null;
-    var tableContainer = tableBlock.querySelector<HTMLElement>(':scope > div.bocListTableContainer')!;
+    const tableContainer = tableBlock.querySelector<HTMLElement>(':scope > div.bocListTableContainer');
+    if (!tableContainer) // In the case of an empty BocList we don't have a table container
+      return;
+
```

Comment:
tableContainer is now of type `HTMLElement?`, but `BocList.FixHeaderPosition` expects the non-nullable `HTMLElement` - see Build Log.

Reply:
Yes this was an initial problem that should be fixed now with the change from `var` to `const`. Since the variable is constant, its type gets "changed" to `HTMLElement` after the null check in the next line. I am re-running the build as it failed because of an external issue

Reply:
Ah, I see.